app/src/main/java/org/p2p/wallet/common/feature_toggles/toggles/remote/NewBuyFeatureToggle.kt

```diff
@@ -0,0 +1,14 @@
+package org.p2p.wallet.common.feature_toggles.toggles.remote
+
+import org.p2p.wallet.BuildConfig
+import org.p2p.wallet.common.feature_toggles.remote_config.RemoteConfigValuesProvider
+
+class NewBuyFeatureToggle(
+    valuesProvider: RemoteConfigValuesProvider
+) : BooleanFeatureToggle(valuesProvider) {
+    override val featureKey: String = "buy_feature"
+    override val featureDescription: String = "Enable new buy feature"
+    override val defaultValue: Boolean = false
+    override val value: Boolean
+        get() = super.value && BuildConfig.NEW_BUY_ENABLED
```

Comment:
do we need BuildConfig field for this feature toggle?

Reply:
i do not know i done this from example
we can get rid of it?

Reply:
yeah, sure

Reply:
ssl pinning is just a little bit different, sorry for inconvenience